### ISSUE TITLE
Output issue on node 5

### ISSUE THREAD

**Issue by MarkHe1222** [anarchy9388/scan#40]
```
Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 610975    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611012    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611050    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611090    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611136    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611180    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611232    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611286    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611341    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611397    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611451    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611505    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611558    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611611    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611667    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611723    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611775    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611828    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611879    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611922    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 61Total parts to preTotal parts to pre-check: 2714880    Processed Parts: 895756    Received: 107    Target Blocks Found: 0    Running Candidate Block Scanner processes:70    Bytes Processed: 68.34 TB
```

**Comment by MarkHe1222** [comment]
Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 610975    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611012    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611050    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611090    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611136    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611180    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611232    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611286    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611341    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611397    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611451    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611505    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611558    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611611    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611667    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611723    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611775    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611828    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611879    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 611922    Received: 75    Target Blocks Found: 0    Running Candidate Block ScaTotal parts to pre-check: 2714880    Processed Parts: 61Total parts to preTotal parts to pre-check: 2714880    Processed Parts: 895756    Received: 107    Target Blocks Found: 0    Running Candidate Block Scanner processes:70    Bytes Processed: 68.34 TB

**Comment by anarchy9388** [comment]
@MarkHe1222  Fixed in 0.9 with shutil
